.github/workflows/build.yml

```diff
@@ -7,7 +7,8 @@ env:
   COUCH_NODE_NAME: nonode@nohost
   BUILDS_SERVER: _couch/builds_testing
   STAGING_SERVER: _couch/builds
-  MARKET_URL: ${{ secrets.MARKET_URL }}
+  AUTH_MARKET_URL: ${{ secrets.MARKET_URL }}
```

Comment:
I think we need to change the name of the GH secret, not necessarily the name of the environment variable. My concern is that branches that don't have this change will still access the `${{ secrets.MARKET_URL }}` and potentially still log it somewhere, and obviously we don't want to leak the _new_ password.

I think you can revert a lot of the changes below by switching the env var name back, and changing the secret name instead.

Reply:
Gotcha, I changed it to what I assume is a good name and reverted the other changes.